docs/upgrade/services_removal.md

```diff
@@ -67,6 +73,7 @@ Following classes have been removed:
 - `OrderService`
 - `OrderStatusService`
 - `FlagService`
+- `AdvancedSearchService`
```

Comment:
you missed `AdvancedSearchOrderService` that has been removed as well :slightly_smiling_face: